lib/contracts/accountParser.js

```diff
@@ -0,0 +1,65 @@
+const bip39 = require("bip39");
+const hdkey = require('ethereumjs-wallet/hdkey');
+const fs = require('../core/fs');
+
+class AccountParser {
+  static parseAccountsConfig(accountsConfig, web3, logger) {
+    let accounts = [];
+    if (accountsConfig && accountsConfig.length) {
+      accountsConfig.forEach(accountConfig => {
+        const account = AccountParser.getAccount(accountConfig, web3, logger);
+        if (!account) {
+          return;
+        }
+        if (Array.isArray(account)) {
+          accounts = accounts.concat(account);
+          return;
+        }
+        accounts.push(account);
+      });
+    }
+    return accounts;
+  }
+
+  static getAccount(accountConfig, web3, logger) {
+    if (!logger) {
+      logger = console;
+    }
+    if (accountConfig.privateKey) {
+      if (!accountConfig.privateKey.startsWith('0x')) {
```

Comment:
I'm not sure if you should need this? Think there should be a formatting util for this in the web3 libs. Good looking out just in case though.

Reply:
consider using this: https://web3js.readthedocs.io/en/1.0/web3-utils.html#isaddress

Reply:
is not about the validation, it simply makes sure it supports with and without the 0x. should avoid user headaches

Reply:
Indeed. I did add the validation underneath because it makes sense and geves a warning to the user if the key is not valid